openmm_md/plain_md.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<p>"We'll use an OpenMM-based <code>PlainMDProtocol</code>"</p><p> </p><p>Maybe it's just me but that sentence feels odd. It somehow feels a little bit redundant or maybe introduced too late? Maybe the wording here should be "here the OpenMM-based MD Protocol is named <code>PlainMDProtocol</code>"?</p><p> </p><p>"The actual simulation is performed by a <code>Protocol</code>"</p><p> </p><p>Would it be worth making the "Protocol" text link to this? <a href="https://docs.openfree.energy/en/stable/guide/models/execution.html#protocols-and-the-execution-model" target="_blank">https://docs.openfree.energy/en/stable/guide/models/execution.html#protocols-and-the-execution-model</a></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>